src/workflow/ArcanistDiffWorkflow.php

```diff
@@ -477,6 +485,31 @@ public function run() {
 
     if (!$this->shouldOnlyCreateDiff()) {
       $revision = $this->buildRevisionFromCommitMessage($commit_message);
+      // UBER CODE
```

Comment:
Move this code to a separate class if possible. Or at least to a separate method.